tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/settings_form.py

```diff
@@ -35,18 +36,13 @@ def __init__(self, marionette):
         Base.__init__(self, marionette)
         view = self.marionette.find_element(*self._settings_view_locator)
         Wait(self.marionette).until(lambda m: view.location['y'] == 0)
```

Comment:
I guess while you're here, you could change this to use view.rect as well, instead of view.location.